src/collective/cover/tests/cover.robot

```diff
@@ -71,12 +71,10 @@ Delete
     Click Button  Delete
     Page Should Contain  Plone site
 
-Edit Cover Layout
-    [Documentation]  Click on Layout tab and wait until the layout has been
-    ...              loaded. Buttons related with layout operations must be
-    ...              also visible.
+Open Layout Tab
```

Comment:
Just out of curiosity, why the name change from "Edit Cover Layout" to "Open Layout Tab"?Just better naming to the changes you made like `Wait Until Page Contains`?